lib/napa/logger/configuration.rb

```diff
@@ -0,0 +1,36 @@
+module Napa
+  class Logger
+    class Configuration
+
+      def initialize(options = {})
+        @options = {}.tap do |o|
+          o[:format] = :basic if Napa.env.development?
+          o[:output] = [:stdout] if ENV['DYNO']
+        end
+
+        @options.merge!(options)
+      end
+
+      def format
+        # Allowed options: :basic, :yaml, :json
+        @options[:format] || :json
```

Comment:
I thought about this and actually started working something up, but it adds a fair level of complexity that we probably don't need right now, so I decided to punt. I could see a potential use case for this in the future, but for now we probably don't need it.